charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,36 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+{{- if .Values.presto.spec.hive.metastore.config.tls.enabled }}
+      - name: ghostunnel-server
+        env:
+        - name: HIVE_METASTORE_CA_CRTFILE
+          value: opt/hive/tls/ca.crt
+        - name: HIVE_METASTORE_KEYFILE
+          value: opt/hive/tls/tls.key
+        - name: HIVE_METASTORE_CRTFILE
+          value: opt/hive/tls/tls.crt
+        command: ["ghostunnel"]
+        args:
+        - server
+        - --listen
+        - localhost:8443
+        - --target
+        - localhost:9083
+        - --key
+        -  HIVE_METASTORE_KEYFILE
```

Comment:
Env vars need $, and to actually have it evaluated by Kubernetes, we need https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config